outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "zone_name" {
+  value       = local.zone_name
+  description = "interpolated value of `local.zone_name`"
+}
+
+output "domain_proof" {
+  value       = aws_route53_record.domain_proof.name
+  description = "interpolated value of `aws_route53_record.domain_proof.name`"
```

Comment:
there seems to be a mismatch with documentation here

Reply:
I agree; I'm now using `segmentio/terraform-docs` to prevent these issues going forward